--- isam/Makefile.in~	Sat Mar 15 10:48:07 2003
+++ isam/Makefile.in	Mon Apr  7 22:59:53 2003
@@ -306,7 +306,8 @@
 LTCOMPILE = $(LIBTOOL) --mode=compile $(CC) $(DEFS) $(DEFAULT_INCLUDES) \
 	$(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS)
 CCLD = $(CC)
-LINK = $(LIBTOOL) --mode=link $(CCLD) $(AM_CFLAGS) $(CFLAGS) \
+CXXLD = $(CXX)
+LINK = $(LIBTOOL) --mode=link $(CXXLD) $(AM_CFLAGS) $(CFLAGS) \
 	$(AM_LDFLAGS) $(LDFLAGS) -o $@
 DIST_SOURCES = $(libnisam_a_SOURCES) $(isamchk_SOURCES) isamlog.c \
 	pack_isam.c test1.c test2.c test3.c
--- myisam/Makefile.in~	Sat Mar 15 10:48:08 2003
+++ myisam/Makefile.in	Mon Apr  7 23:11:44 2003
@@ -339,7 +339,8 @@
 LTCOMPILE = $(LIBTOOL) --mode=compile $(CC) $(DEFS) $(DEFAULT_INCLUDES) \
 	$(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS)
 CCLD = $(CC)
-LINK = $(LIBTOOL) --mode=link $(CCLD) $(AM_CFLAGS) $(CFLAGS) \
+CXXLD = $(CXX)
+LINK = $(LIBTOOL) --mode=link $(CXXLD) $(AM_CFLAGS) $(CFLAGS) \
 	$(AM_LDFLAGS) $(LDFLAGS) -o $@
 DIST_SOURCES = $(libmyisam_a_SOURCES) ft_dump.c mi_test1.c mi_test2.c \
 	mi_test3.c myisamchk.c myisamlog.c myisampack.c
--- libmysqld/examples/Makefile.in~	Sat Mar 15 10:48:08 2003
+++ libmysqld/examples/Makefile.in	Tue Apr  8 00:24:40 2003
@@ -274,7 +274,7 @@
 	$(CXXLINK) $(mysql_LDFLAGS) $(mysql_OBJECTS) $(mysql_LDADD) $(LIBS)
 mysqltest$(EXEEXT): $(mysqltest_OBJECTS) $(mysqltest_DEPENDENCIES) 
 	@rm -f mysqltest$(EXEEXT)
-	$(LINK) $(mysqltest_LDFLAGS) $(mysqltest_OBJECTS) $(mysqltest_LDADD) $(LIBS)
+	$(CXXLINK) $(mysqltest_LDFLAGS) $(mysqltest_OBJECTS) $(mysqltest_LDADD) $(LIBS)
 
 mostlyclean-compile:
 	-rm -f *.$(OBJEXT) core *.core
